src/components/Cell/RecipeHeader.js

```diff
@@ -43,40 +51,39 @@ const { Header, Footer, Content } = Layout;
 // <Header style={cssHeader} />
 
 class RecipeHeader extends Component {
-  // shouldComponentUpdate(nextProps) {
-  // if (this.props.title !== nextProps.title || this.props.toggle !== nextProps.toggle) {
-  //   console.log("Name and Id RenderFooter **********************");
-  //       return true;
-  //     }
-  //     console.log("False RenderFooter****************************");
-  //     return false;
-  //   }
-  render(){
-    return(
-      <Header style={{
-        // background: '#fff',
-        // color:'red',
-        //padding: 0
-        fontSize: '20px'
-    }}>
-        <Row>
-          <Col span={20}>
-            "{this.props.title}" Template
-          </Col>
-          <Col span={4}>
-
-            <Icon
-              className="tile-close"
-              type="close"
-              style={{ cursor: "pointer", transition: "color .3s" }}
-              onClick={this.props.toggle}
-            />
-
-          </Col>
-        </Row>
-
-      </Header>
-    )
-  }
+	// shouldComponentUpdate(nextProps) {
+	// if (this.props.title !== nextProps.title || this.props.toggle !== nextProps.toggle) {
+	//   console.log("Name and Id RenderFooter **********************");
+	//       return true;
+	//     }
+	//     console.log("False RenderFooter****************************");
+	//     return false;
+	//   }
+	render() {
+		return (
+  <Header className='RecipeHeader-header'>
```

Comment:
this part is non formatted well, please update it

Reply:
does class RecipeHeader get updated styles? i mean as we removed a scss file? not sure...
and your approach of naming classes looks like BEM style. it's bad(not critical, but..). Details: https://medium.com/@bluebirdyourdestiny/bem-should-not-exist-6414005765d6

Reply:
Oooops , I have resolved it now.

Reply:
cool - please add it to this PR

Reply:
and sync your fork, because we have conflicts